### PR TITLE
[zh] sync v1.24 for api-reference: cluster-role-binding-v1.md

### PR DESCRIPTION
**Pull request by windsonsea**
Accepted localization task from issue #33798 and triaged by @tengqm.  This is refer-1 for submission.

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 1]
```diff
@@ -0,0 +1,651 @@
...
+`import "k8s.io/api/rbac/v1"`
+<!--
+## ClusterRoleBinding {#ClusterRoleBinding}
+
+ClusterRoleBinding references a ClusterRole, but not contain it.  It can reference a ClusterRole in the global namespace, and adds who information via Subject.
+-->
+## ClusterRoleBinding {#ClusterRoleBinding}
+
+ClusterRoleBinding 引用 ClusterRole，但不包含它。
+它可以在全局命名空间中引用，并通过 Subject 添加 who 信息。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:43]:
```suggestion
它可以引用全局命名空间中的 ClusterRole，并通过 Subject 添加主体信息。
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+    APIGroup is the group for the resource being referenced
+  - **roleRef.kind** (string), required
+    Kind is the type of resource being referenced
+  - **roleRef.name** (string), required
+    Name is the name of resource being referenced
+-->
+
+- **roleRef** (RoleRef)，必选
+  
+  RoleRef 只能在全局命名空间中引用 ClusterRole。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:78]:
```suggestion
  RoleRef 只能引用全局命名空间中的 ClusterRole。
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 3]
```diff
@@ -0,0 +1,651 @@
...
+  如果无法解析 RoleRef，则 Authorizer 必定返回一个错误。
+  
+  <a name="RoleRef"></a>
+  *RoleRef 包含指向正被使用的角色的信息。*
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:82]:
```suggestion
  **RoleRef 包含指向正被使用的角色的信息。**
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 4]
```diff
@@ -0,0 +1,651 @@
...
+  
+  - **roleRef.apiGroup** (string)，必选
+    
+    APIGroup 是被引用资源的组
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:86]:
```suggestion
    apiGroup 是被引用资源的组
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 5]
```diff
@@ -0,0 +1,651 @@
...
+  
+  - **roleRef.kind** (string)，必选
+    
+    Kind 是被引用的资源的类型
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:90]: 类似这种，要用字段名称的正确大小写，原来生成的文档不准确。

```suggestion
    kind 是被引用的资源的类型
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 6]
```diff
@@ -0,0 +1,651 @@
...
+  
+  - **roleRef.name** (string)，必选
+    
+    Name 是被引用的资源的名称
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:94]: 下同。
```suggestion
    name 是被引用的资源的名称
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 7]
```diff
@@ -0,0 +1,651 @@
...
+<!-- 
+- **subjects** ([]Subject)
+  Subjects holds references to the objects the role applies to.
+  <a name="Subject"></a>
+  *Subject contains a reference to the object or user identities a role binding applies to.  This can either hold a direct API object reference, or a value for non-objects such as user and group names.*
+-->
+- **subjects** ([]Subject)
+  
+  Subjects 包含角色所应用到的对象的引用。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:103]:
```suggestion
  Subjects 包含角色所适用的对象的引用。
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 8]
```diff
@@ -0,0 +1,651 @@
...
+  
+  <a name="Subject"></a>
+  *Subject 包含角色绑定所应用到的对象或用户标识的引用。这可以包含直接 API 对象的引用或非对象（如用户名和组名）的值。*
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:106]:
```suggestion
  **Subject 包含对角色绑定所适用的对象或用户标识的引用。其中可以包含直接 API 对象的引用或非对象（如用户名和组名）的值。**
```

Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:106]:
```suggestion
  **Subject 包含对角色绑定所适用的对象或用户标识的引用。其中可以包含直接 API 对象的引用或非对象（如用户名和组名）的值。**
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 9]
```diff
@@ -0,0 +1,651 @@
...
+  <!--
+  - **subjects.kind** (string), required
+    Kind of object being referenced. Values defined by this API group are "User", "Group", and "ServiceAccount". If the Authorizer does not recognized the kind value, the Authorizer should report an error.
+  - **subjects.name** (string), required
+    Name of the object being referenced. 
+  -->
+  - **subjects.kind** (string)，必选
+    
+    被引用的对象的种类。这个 API 组定义的值是 `User`、`Group` 和 `ServiceAccount`。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:115]: kind 统一翻译为“类别”。

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 10]
```diff
@@ -0,0 +1,651 @@
...
+    如果 Authorizer 未识别种类值，则 Authorizer 应报告一个错误。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:116]:
```suggestion
    如果 Authorizer 无法识别种类值，则 Authorizer 应报告一个错误。
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 11]
```diff
@@ -0,0 +1,651 @@
...
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+- **propagationPolicy** (*in query*): string
+  <a href="{{< ref "../common-parameters/common-parameters#propagationPolicy" >}}">propagationPolicy</a>
+- **resourceVersion** (*in query*): string
+  <a href="{{< ref "../common-parameters/common-parameters#resourceVersion" >}}">resourceVersion</a>
+- **resourceVersionMatch** (*in query*): string
+  <a href="{{< ref "../common-parameters/common-parameters#resourceVersionMatch" >}}">resourceVersionMatch</a>
+- **timeoutSeconds** (*in query*): integer
+  <a href="{{< ref "../common-parameters/common-parameters#timeoutSeconds" >}}">timeoutSeconds</a>
+-->
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:594]: If it's no need to translate, I think the comment part can be ignored.

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 12]
```diff
@@ -0,0 +1,651 @@
...
+  import: "k8s.io/api/rbac/v1"
+  kind: "ClusterRoleBinding"
+content_type: "api_reference"
+description: "ClusterRoleBinding 引用 ClusterRole，但不包含它。"
+title: "ClusterRoleBinding"
+weight: 6
+auto_generated: true
+---
+
+<!--
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:13]: There are many trailing spaces in this file, please check.

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 13]
```diff
@@ -0,0 +1,651 @@
...
+
+<hr>
+-->
+## 操作 {#Operations}
+
+<hr>
+<!-- 
+### `get` read the specified ClusterRoleBinding
+#### HTTP Request
+GET /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/{name}
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:181]: This line is not (and no need to) translate, maybe it's better to remove from comment.

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 14]
```diff
@@ -0,0 +1,651 @@
...
+401: Unauthorized
+<!-- 
+### `list` list or watch objects of kind ClusterRoleBinding
+
+#### HTTP Request
+
+GET /apis/rbac.authorization.k8s.io/v1/clusterrolebindings
+
+#### Parameters
+-->
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:225]: I think `#### Parameters` is better to be grouped with next paragraph or standalone.

**Review by Sea-n — COMMENTED** [review]
Thanks for your translation, here's some formatting nitpick.

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md (hunk 12 above)
Comment (windsonsea) [reply content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:12]: hmmm, removed much. zh followed the en source and has many trailing spaces too -_!

**Review by windsonsea — COMMENTED** [review]
only comments feedback

**Review by windsonsea — COMMENTED** [review]
comments feedback

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 15]
```diff
@@ -0,0 +1,486 @@
...
+    Name is the name of resource being referenced
+-->
+- **roleRef** (RoleRef)，必需
+  RoleRef 只能引用全局命名空间中的 ClusterRole。
+  如果无法解析 RoleRef，则 Authorizer 必定返回一个错误。
+  
+  <a name="RoleRef"></a>
+  **RoleRef 包含指向正被使用的角色的信息。**
+  - **roleRef.apiGroup** (string)，必需
+    apiGroup 是被引用资源的组
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:72]: 中文里的空行还是要保留的，目的是标识独立段落，不然此行将与上一行合并渲染。

下同。

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 16]
```diff
@@ -0,0 +1,486 @@
...
+<!--
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+  Standard object's metadata.
+- **items** ([]<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>), required
+  Items is a list of ClusterRoleBindings
+-->
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+  标准的对象元数据。
+- **items** ([]<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>)，必需
+  Items 是 ClusterRoleBindings 的列表。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:135]:
```suggestion
  items 是 ClusterRoleBindings 的列表。
```

File: content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 17]
```diff
@@ -0,0 +1,486 @@
...
+  name of the ClusterRoleBinding
+- **pretty** (*in query*): string
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+-->
+
+#### 参数
+- **name** (**路径参数**): string，必需
+  ClusterRoleBinding 的名称
+- **pretty** (**查询参数**): string
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:162]: 以上各行之间要加空行。

Comment (windsonsea) [reply content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md]: 好的，明哥，有预览页面了吗？我对着 preview 看会比较直观 ：）

Comment (tengqm) [reply content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md]: 现在看不到。

**Pull request merged**